selenium_tests/basic_test.py

```diff
@@ -32,11 +32,13 @@ def test_zero_price_purchase(self):
 
         # Click the Enroll Now button on dashboard
         self.wait().click(lambda driver: driver.find_element_by_class_name("enroll-button"))
+        self.wait().click(lambda driver: driver.find_element_by_class_name("pay-button"))
```

Comment:
Oh yeah, initially we wanted to show a disabled 'enroll now' button when there were no future course runs, but we changed the design to look like this:

![asdjfjfjf](https://cloud.githubusercontent.com/assets/6207644/26788996/eba76638-49dc-11e7-8478-e02439340325.png)

which doesn't include the button. the `disabled` prop was something I added to support the previous design (which included the disabled button) but I forgot to remove it. I'll take it out now.